test/tmpMigration.test.ts

```diff
@@ -10,33 +9,27 @@ dayjs.extend(utc);
 dayjs.extend(timezone);
 
 async function tmpMigration() {
-  //   console.log(
-  //     await prisma.$queryRaw(`
-  //   select
-  //   id as "userId",
-  //   CAST(CONCAT(CAST(("startTime") AS text), ' minute')::interval AS time) as "startTime",
-  //   CAST(CONCAT(CAST(("endTime") AS text), ' minute')::interval AS time)  as "endTime",
-  //   ARRAY [0,1,2,3,4,5,6]
-  // from
-  //   (
-  //     select
-  //       users.id,
-  //       users."startTime",
-  //       users."endTime",
-  //       users."timeZone",
-  //       count("Availability".id) as availability_count
-  //     from users
-  //     left join "Availability" on "Availability"."userId" = users.id
-  //     group by users.id
-  //   ) usersWithAvailabilityNumber
-  // where availability_count < 1
-  // `)
-  //   );
-  const MIGRATION_SQL = fs
-    .readFileSync(__dirname + "/../prisma/migrations/20211115182559_availability_issue/migration.sql")
-    .toString();
-  await prisma.$queryRaw(MIGRATION_SQL);
-
+  await prisma.$queryRaw`
```

Comment:
I thought this migration has already been ran? 🤔

Reply:
Yeah, this is a test which was using the migration sql you speak of. The test still exists, so I made changes to ensure that this test doesn't fail because of the updated queryRaw syntax

Reply:
Although, I'm not sure if we still need this test

Reply:
We may not need this anymore. @KATT could give us more info on this (but he's on vacation) So I say let's remove it. We can always revert later if needed.

Reply:
Okay. Removing it for now